frontend/src/components/Signup.js

```diff
@@ -18,40 +118,151 @@ export default function Signup({changeToFalse}) {
+              helperText={
+                lastNameError ? "Please enter a valid last name." : ""
+              }
+              inputProps={{ style: { color: "black" } }}
+              color="secondary"
+              margin="normal"
+              className="register-text signup-text"
+              label="Last Name"
+              variant="outlined"
+              value={lastName}
+              onChange={(e) => {
+                setLastName(e.target.value);
+                validateLastName(e.target.value);
+              }}
+            />
+            <TextField
+              error={phoneNumberError}
+              helperText={
+                phoneNumberError ? "Please enter a valid phone number." : ""
+              }
+              inputProps={{ style: { color: "black" } }}
+              color="secondary"
+              margin="normal"
+              className="register-text signup-text"
+              label="Phone Number"
+              variant="outlined"
+              required
+              value={phoneNumber}
+              onChange={(e) => {
+                setPhoneNumber(e.target.value);
+                validatePhoneNumber(e.target.value);
+              }}
+            />
+            <TextField
+              error={emailError}
+              helperText={emailError ? "Please enter a valid email." : ""}
+              inputProps={{ style: { color: "black" } }}
+              color="secondary"
+              margin="normal"
+              className="register-text signup-text"
+              label="Email Address"
+              variant="outlined"
+              required
+              value={email}
+              onChange={(e) => {
+                setEmail(e.target.value);
+                validateEmail(e.target.value);
+              }}
+            />
+            <TextField
+              error={passwordError}
+              helperText={
+                passwordError
+                  ? "Password should have at least one digit, lowercase letter, uppercase letter, special character, and is at least 8 characters long"
+                  : ""
+              }
+              inputProps={{ style: { color: "black" } }}
+              color="secondary"
+              margin="normal"
+              className="register-text signup-text"
+              label="Password"
+              variant="outlined"
+              type="password"
+              required
+              value={password}
+              onChange={(e) => {
+                setPassword(e.target.value);
+                validatePassword(e.target.value);
+              }}
+            />
+          </div>
+
+          <div className="register-button">
+            <p>{signUpStatusMessage}</p>
+            <Button
+              type="submit"
+              variant="contained"
+              onClick={handleClick}
+              sx={{
+                margin: 1,
+                color: "white",
+                fontSize: "1.2rem",
```

Comment:
please wrap text in the typography and use appropriate variant instead of inline style